src/main/java/edu/cornell/kfs/concur/batch/ConcurStandardAccountingExtractToPdpAndCollectorStep.java

```diff
@@ -22,7 +24,7 @@ public boolean execute(String jobName, Date jobRunDate) throws InterruptedExcept
         List<String> listOfSaeFileNames = getConcurStandardAccountingExtractService().buildListOfFileNamesToBeProcessed();
         boolean success = true;
```

Comment:
Jay, I don't see a change or comment regarding this matter, I wonder if it got missed.